csharp/src/Ice/IObject.cs

```diff
@@ -13,10 +14,15 @@ public interface IObject
         /// <summary>Dispatches a request on this servant.</summary>
         /// <param name="request">The <see cref="IncomingRequestFrame"/> to dispatch.</param>
         /// <param name="current">Holds decoded header data and other information about the current request.</param>
+        /// <param name="cancel">A cancellation token that is notified of cancellation when the dispatch is cancelled.
```

Comment:
I would use the American spelling for canceled, with one L.

Reply:
ok